.eslintrc

```diff
@@ -1,15 +1,19 @@
 {
   "globals": {
-    "describe": true,
-    "it": true,
-    "xit": true,
-    "beforeEach": true,
-    "afterEach": true,
-    "context": true,
-    "chai": true,
-    "expect": true,
-    "assert": true,
-    "sinon": true,
+    "describe": false,
+    "it": false,
+    "xit": false,
```

Comment:
Can the test-specific globals be pulled out into their own eslint file? `describe` is a valid global in tests, but not in user-facing code.